### PR TITLE
feat(curriculum): add 1.2 EC animations

### PR DESCRIPTION
**Pull request by DanielRosa74**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e421f8d8e9871a83cda7f.md
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e421f8d8e9871a83cda7f.md:1]: The first of the sentence is missing in the audio.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e42d9cf8693725896cc49.md
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e42d9cf8693725896cc49.md:1]: Remove the audio.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e4b7d381d567e8d97967c.md
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e4b7d381d567e8d97967c.md:1]: Bob is not showing up

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e4c1f0bd3e97ef2d46644.md
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e4c1f0bd3e97ef2d46644.md:1]: Bob is not showing up

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e4cd7f87d4f7f6954446d.md
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e4cd7f87d4f7f6954446d.md:1]: Bob again

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e523fcaaa0e823b3789e5.md
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e523fcaaa0e823b3789e5.md:1]: Remove the audiopath

**Review by nieldakarla — CHANGES_REQUESTED** [review]
That is a beautiful work, Daniel! Congratulations! =D 
I've added a few corrections to the challenges and found some audioPaths that need to be deletd in dialogue 4 and 5.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657b5bd17d068521ee895631.md
[the remark this answers is not in the document]
Comment (DanielRosa74) [reply curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657b5bd17d068521ee895631.md:1]: Changed to third person plural (their)

**Comment by DanielRosa74** [comment]
Hi, [nieldakarla](https://github.com/nieldakarla). Requested changes added! Thanks a lot!

**Review by nieldakarla — APPROVED** [review]
LGTM

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**